src/main/java/net/openhft/chronicle/queue/reader/ChronicleHistoryReader.java

```diff
@@ -225,7 +225,7 @@ protected WireParselet parselet() {
     protected void processMessage(CharSequence methodName, MessageHistory history) {
         CharSequence extraHistoId = histosByMethod ? ("_" + methodName) : "";
         long lastTime = 0;
-        // if the tailer has recordHistory(true) then the MessageHistory will be
+        // if the tailer has recordHistory (sourceId != 0) then the MessageHistory will be
```

Comment:
should we be removing `recordHistory` altogether here?

Reply:
There is still a getter for recordHistory() (it returns sourceId() != 0)